en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -33,66 +31,58 @@ To detect anomalies of the service, you must trace its executions from the code
     
-- **View log entries**
-
-    If you click on the required area on the graph, the log entries applicable for that time period are displayed in the **Logs** pane. In this example, the log displays multiple entries indicating an error in connecting to the `hr-service`.
-    
-- **View data for a custom time range**
-
-    You can expand the graph by clicking and dragging the pointer over the period you want to drill down. The default graph displays data for a custom time range where the data points are at a lower granularity.
+- **View log entries:**
+  If you click the required area on the graph, the log entries applicable for that period are displayed in the **Logs** pane. In this example, the log displays multiple entries indicating an error in connecting to the `hr-service`.
+     
+- **View data for a custom time range:**
+  You can expand the graph by clicking and dragging the pointer over the period you want to drill down. The default graph displays data for a custom time range where the data points are at a lower granularity.
 
  
 ### Analyze the latency graph
 
 The latency graph depicts the latency of requests over a selected time interval. You can analyze the spikes in the latency graph to identify the API invocations that result in error status.
-
 The actions you can perform are as follows:
 
-- **View details of a latency that has occurred**
-
-    To do this, go to the latency graph under the **Throughput & Latency** tab and click on a data point where there is a spike. This displays the latency, start time, and status of the corresponding requests just above the low-code diagram as shown in the image below:
+- **View details of a latency that has occurred:**
+  To do this, go to the latency graph under the **Throughput & Latency** tab and click a data point where there is a spike. This displays the latency, start time, and status of the corresponding requests just above the low-code diagram as shown in the following figure:
 
     ![View latency details](../assets/img/observability/latency-details.png)
     
-- **View details at a higher granularity**
+- **View details at a higher granularity:**
    
     If you want to drill down to view details at a higher granularity, drag the pointer over a time range to view the graph for the selected period.
     
-- **Check the status code of a request that has caused a latency**
+- **Check the status code of a request that has caused a latency:**
 
-    To do this, click on the relevant latency value. In the following example, the status code is `500`, which means the request resulted in an error state.
+    To do this, click the relevant latency value. In the following figure, the status code is `500`, which means the request resulted in an error state:
```

Comment:
should this be error state or erroneous state?

Reply:
Both seem correct. Therefore, did not go ahead and change.